lib/ts/recipe/emailpassword/components/themes/resetPasswordUsingToken/resetPasswordEmail.tsx

```diff
@@ -25,48 +25,44 @@ import GeneralError from "../../library/generalError";
 
 import type { EnterEmailProps, EnterEmailStatus } from "../../../types";
 
-const EmailPasswordResetPasswordEmail: React.FC<EnterEmailProps> = (props) => {
-    const t = useTranslation();
-    const userContext = useUserContext();
-    const [status, setStatus] = useState<EnterEmailStatus>("READY");
-    const [emailFieldValue, setEmailFieldValue] = useState<string>("");
-
-    const onSuccess = (): void => {
-        setStatus("SENT");
-    };
-
-    const resend = (): void => {
-        setStatus("READY");
-    };
-    const { formFields } = props;
+const EmailPasswordResetPasswordEmail = withOverride(
```

Comment:
Please export this separately and use it in the override definitions https://github.com/AmanAgarwal041/supertokens-auth-react/blob/bfe9f61d956c070ca603b88db241069c50af1477/lib/ts/recipe/emailpassword/types.ts#L46

Reply:
Done